db/migrate/20230919171611_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, foreign_key: { to_table: :users }
```

Comment:
- [ ] The foreign key between the comments table and the users table should be `user_id`, not `author_id` as you did. Please fix that. Check the given ERD below.

![image](https://github.com/alexmedinasf/Blog/assets/68796232/a7b48ce4-a715-4565-aa8f-50a9a35e18bf)

[Hint] It's supposed to be like that:
```ruby
t.references :user, foreign_key: true
```